src/Content/OEmbed.php

```diff
@@ -86,8 +86,7 @@ public static function fetchURL($embedurl, $no_rich_type = false)
 				$redirects = 0;
 				$html_text = Network::fetchUrl($embedurl, false, $redirects, 15, 'text/*');
 				if ($html_text) {
-					$dom = new DOMDocument();
-					$dom->loadHTML($html_text);
+					$dom = @DOMDocument::loadHTML($html_text);
```

Comment:
This isn't a static method, what was the fatal error you fixed with this?

Reply:
I got massive warnings because of malformatted HTML - so I reverted the code here.

Reply:
No. This had been my first move.

Reply:
Drat.